mirgecom/viscous.py

```diff
@@ -275,63 +277,170 @@ def viscous_flux(state, grad_cv, grad_t):
             momentum=tau, species_mass=-j)
 
 
-def viscous_facial_flux(discr, state_tpair, grad_cv_tpair, grad_t_tpair,
-                        local=False):
-    """Return the viscous flux across a face given the solution on both sides.
+def viscous_divergence_flux(discr, state_pair, grad_cv_pair, grad_t_pair,
```

Comment:
```suggestion
def viscous_facial_flux(discr, state_pair, grad_cv_pair, grad_t_pair,
```
(for symmetry with inviscid.)

Also, maybe hedge and call it `viscous_facial_flux_central`? (I thought it was already called this at one point...?) Unless we're sure we're not going to add any more viscous flux types (in which we case we don't need the `numerical_flux_func` argument in `viscous_flux_on_element_boundary`).

Reply:
Yes we have planned on using different viscous fluxes - esp when we were examining the BCs closely and matching our implementation with Mengaldo.